.github/workflows/backend-test.yml

```diff
@@ -0,0 +1,95 @@
+name: Backend Test
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  misc:
+    runs-on: ubuntu-latest
+    env:
+      CMAKE_DUMMY_WEBUI: true
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - uses: actions/setup-python@v2
+      - run: python -m pip install -r rst/requirements.txt
+
+      - name: Install tarantool
+        run: |
+          curl -L https://tarantool.io/installer.sh | sudo VER=1.10 bash
+
+      - name: Install ldoc
+        run: >
+          tarantoolctl rocks install --server=http://rocks.moonscript.org
+          https://raw.githubusercontent.com/tarantool/ldoc/tarantool/ldoc-scm-2.rockspec
+
+      - name: Build cartridge
+        env:
+          CMAKE_LDOC_FIND_REQUIRED: 'YES'
+          CMAKE_SPHINX_FIND_REQUIRED: 'YES'
+        run: |
+          tarantoolctl rocks make
```

Comment:
Why not extract this into `build` job and depend on it from other jobs?

- it won't duplicate
- it won't build twice for each new ref

Reply:
Creating another job would require reinstalling tarantool with all deps and passing an artifact between jobs. It's not worth single-second `rocks make`.

Reply:
ok. If it's not really saves time, then nevermind.